test/pkg/environment/aws/environment.go

```diff
@@ -90,3 +91,15 @@ func GetTimeStreamAPI(session *session.Session) timestreamwriteiface.TimestreamW
 	}
 	return &NoOpTimeStreamAPI{}
 }
+
+func GetClusterName() string {
+	return lo.Must(os.LookupEnv("CLUSTER_NAME"))
```

Comment:
Why do we need to re-get this value every time? Why not get it at the beginning of the test suite during the setup in the `test/pkg/environment/aws` Environment struct and then store it in the environment so users can get these values with `env.ClusterName` or `env.ClusterEndpoint`

Reply:
There's not really any reason not to, I can go ahead and change that.